core/internal/hashencode/hashencode_test.go

```diff
@@ -33,11 +34,11 @@ func TestHashValidity(t *testing.T) {
 func TestVerifyFileB64MD5(t *testing.T) {
 	testFile, err := os.CreateTemp("", "")
 	assert.NoError(t, err)
-	defer func() {
-		_ = testFile.Close()
-	}()
+	defer testFile.Close()
+	defer os.Remove(testFile.Name())
```

Comment:
the defer order is last-in-first-out, so i think you should modify the order or you can put both in a deferred anon function.